alphagov/di-account-management-frontend
# OLH-1113: Contact centre triage page

## Proposed changes
Build a static version of the triage page using the content and designs from [the prototype](https://di-contact-centre-2dd92a1953cd.herokuapp.com/exploratory-concepts/v1/contact-triage#). 

We don’t yet have final content for the page or Welsh translations so the page will be all in English for now. 
The links on the page aren't working yet either as it's not fully clear what they should be.

The value of the reference code is also hard coded for now. 

For now the webchat section, phone contact section, and guidance section are all behind flags.
The page without guidance links, webchat or phone contact:
<img width="1133" alt="Screenshot 2023-10-06 at 12 52 23" src="https://github.com/alphagov/di-account-management-frontend/assets/7116819/c97183cc-a1f2-4502-b2b4-85b9f0f485e5">

The page with everything switched on:
![localhost_6001_contact-govuk-one-login](https://github.com/alphagov/di-account-management-frontend/assets/7116819/4579d79d-bf22-4aa6-809b-cc15b1cae6b2)



Another detail included in the PR is that a **Sign out** link displays in the header when the user is authenticated with OL.

![Screenshot 2023-10-06 at 13 00 44](https://github.com/alphagov/di-account-management-frontend/assets/7116819/7a387d8f-edec-4a1b-a7c7-e107ec9052b8)



### Related links




## Checklists


### Environment variables or secrets


- [ ] Application configuration is up-to-date
- [ ] Documented in the README
- [ ] Added to deployment steps
- [ ] Added to local startup config

## Testing



## How to review

- Check the content is correct (already demo'ed to UCD and they approved
- Check the flags all work
- Check that the changes to how the account navigation is displayed, and the new Sign out link added to the header for this page, aren't breaking anything
